stories/components/Grid/Grid.docs.tsx

```diff
@@ -0,0 +1,238 @@
+import { useMolecules, withDocsWrapper } from '../../common';
+import { Source } from '@storybook/addon-docs';
+
+const DocsPage = () => {
+    const { View, H1, H2, H3, Text, Code } = useMolecules();
+
+    return (
+        <View>
+            <H1>Grid</H1>
+            <Text>
+                Grid is a component that can be used to create a responsive grid layout. It is a
+                wrapper component that uses <Code>FlatList</Code> to render the columns.
+            </Text>
+            <H2>Props</H2>
+            <Text>
+                The Grid component accepts a Configuration Object with the following properties:{' '}
+                {'\n'}
+            </Text>
+            <H3>
+                <Code>data</Code>
+            </H3>
+            <Text>
+                The data prop is an array of objects that represent a column in the grid. Each
+                object in the array may have the following properties: {'\n\n'}
+                <Code>breakPoints</Code> - A number or an object that represents the number of
+                columns that the column should span for each screen breakpoint. {'\n\n'}
+                <Code>theme</Code> - An id of a theme that will be applied to the column. {'\n\n'}
+                <Code>name</Code> - The name of the module or field to render. {'\n\n'}
+                <Code>alignment</Code> - The alignment of the column.
+            </Text>
+            <H3>
+                <Code>renderer</Code>
+            </H3>
+            <Text>
+                The renderer is a component that will be used to render the module or field in each
+                column. It accepts a single prop called <Code>name</Code> that represents the name
+                of the module or field to render. {'\n\n'}
+            </Text>
+
+            <H2>Usage</H2>
+            <Text>
+                Create the <Code>renderer</Code> function that will be passed to the{' '}
+                <Code>Grid</Code> component. For example:
+                <Source language="tsx" code={firstCodeBlock} />
+                Then, use the <Code>Grid</Code> component to render the grid layout:
+                <Source language="tsx" code={secondCodeBlock} />
+            </Text>
+        </View>
+    );
+};
+
+const firstCodeBlock = `
+import { memo, useCallback } from 'react';
+import type { ViewProps } from 'react-native';
+import Grid from '../../../src/components/Grid/Grid';
+import { useMolecules } from '../../../src/hooks';
```

Comment:
import from "@bambooapp/bamboo-molecules"

Reply:
Lets make it a working Demo.

All the components you require, should be created in the file itself.
By that I mean, do not import it from Grid, for the documentation only